packages/nuejs.org/docs/index.md

```diff
@@ -107,7 +107,7 @@ Nue is a perfect system for freelancers and design agencies who want to offer th
 
 
 ### Try it
-You might not believe all the big claims until you [give Nue a try](installation.html). Chances are that you will start wondering why you have ever built websites in any other way.
+You might not believe all these big claims until you [give Nue a try](installation.html). Chances are you will start wondering why you have ever built websites in any other way.
```

Comment:
```suggestion
You might not believe all these big claims until you [give Nue a try](installation.html). Chances are, you will start wondering why you have ever built websites in any other way.
```

Just feels more like I would say it... Feel free to reject.

Reply:
👍